### DISCUSSION TITLE
Filling Font with 1 unique color

### DISCUSSION THREAD

**Discussion by a-darya** [ImageMagick/ImageMagick#3409] [Help] [answered]
Hi,

When filling in fonts (using C++ fillColor() method), a number of unique colors are used - mostly around the edges in order to soften it up. Is there an option to either only use one unique color, or reduce the amount, when initially filling in the color? Trying to avoid using quantization after the fact, because that causes considerable CPU usage in this application.

Thanks!

**Comment by fmw42** [comment, marked as answer]
I do not know C++, but in command line use +antialias to turn off the anti-alias so that you have no smoothing that will cause color blending.

**Reply by a-darya** [reply]
Thank you! Calling the Image::textAntiAlias(false) did the trick.
